x-pack/plugin/transform/qa/multi-cluster-tests-with-security/build.gradle

```diff
@@ -53,16 +54,29 @@ testClusters.register('mixed-cluster') {
 tasks.register('remote-cluster', RestIntegTestTask) {
   mustRunAfter("precommit")
   systemProperty 'tests.rest.suite', 'remote_cluster'
+  maybeDisableForFips(it)
 }
 
 tasks.register('mixed-cluster', RestIntegTestTask) {
   dependsOn 'remote-cluster'
   useCluster remoteCluster
   systemProperty 'tests.rest.suite', 'multi_cluster'
+  maybeDisableForFips(it)
 }
 
 tasks.register("integTest") {
   dependsOn 'mixed-cluster'
+  maybeDisableForFips(it)
 }
 
 tasks.named("check").configure { dependsOn("integTest") }
+
+//TODO: remove with version 8.14. A new FIPS setting was adding in 8.13. Since FIPS configures all test clusters and this specific integTest uses
```

Comment:
```suggestion
//TODO: remove with version 8.14. A new FIPS setting was added in 8.13. Since FIPS configures all test clusters and this specific integTest uses
```